js/background.js

```diff
@@ -46,47 +48,72 @@ chrome.tabs.onActivated.addListener(function(tabId, changeInfo, tab) {
 
 function notifyContainerCreation() {
     // get current tab and send a message to it
-    chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
-      if (!!tabs[0]) {
-        console.info('We hope you\'re having a good time');
-        chrome.tabs.sendMessage(
-            tabs[0].id,
-            {type: createType, text: currentChannel},
-            () => {}
-        )
-      }
+    chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) {
+        if (!!tabs[0]) {
```

Comment:
We're going to need to start preventing the "tabs undefined" errors in the future. I won't refuse the PR but I'll think of a way to set up a dev environment so we can use TypeScript for more safety and quality in our code.